### ISSUE TITLE
stuck in confirming match for 1 hour and if i leave can be punished 

### ISSUE THREAD

**Issue by skulltom666** [ValveSoftware/Dota2-Gameplay#11797]
### Description

confirming match duration standard was 5 seconds no more now queue is like 1 hour and my behavior is good pls fix this

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/106767254/1f70eb84-92ce-4766-bdf3-d7c61f7bf40d)

**Comment by cesiumatom** [comment]
Been having the same issue. Every 2-3 mins it returns to Finding Match for 5-10 seconds before saying Confirming Match, and then it stalls again for minutes. You can only cancel without consequences if you wait for the switch to happen and cancel searching when it says Finding Match. Pretty annoying bug.

**Comment by Nratre** [comment]
Same issue. Never received accept match but got penalised for not being ready after queuing

**Comment by starixmate** [comment]
I HAVE ALSO THE SAME PROBLEM

**Comment by ciandt-crodrigues** [comment]
Same issue here

**Comment by starixmate** [comment]
What we have to do? my queue confirmation is for 2 hours to now?

**Comment by rexablack** [comment]
Same issue here :(

**Comment by merakmarey** [comment]
play LOL

**Comment by Cloudfire8574** [comment]
I too am getting this error. Please work on this.

**Comment by ritzrivastav91** [comment]
Same bug! Stuck in confirming match from last 30 mins

**Comment by j-a-r** [comment]
Yup. Same issue here. Been going on all day and I am unable to join any matches. And I am being punished for "leaving". I have reboot my PC, uninstalled the game completely and re-installed it. Currently on a 45 minute time ban. Sucks. PLEASE FIX THIS ASAP!

**Comment by D41191095** [comment]
Same here...

**Comment by calebphess** [comment]
duplicate issue: https://github.com/ValveSoftware/Dota2-Gameplay/issues/11791

**Comment by quietmkb** [comment]
Same here, it was not as bad earlier today but it got worse now

**Comment by driftaholic** [comment]
Confirming I have the same problem.

**Comment by linrongc** [comment]
check twitch. this is a common issue is valve is not fixing it?
